Extension/src/LanguageServer/settings.ts

```diff
@@ -312,89 +326,93 @@ export class CppSettings extends Settings {
+    public get intelliSenseCachePath(): string | undefined { return changeBlankStringToUndefined(this.getAsStringOrUndefined("intelliSenseCachePath")); }
+    public get intelliSenseCacheSize(): number { return this.getAsNumber("intelliSenseCacheSize"); }
+    public get intelliSenseMemoryLimit(): number { return this.getAsNumber("intelliSenseMemoryLimit"); }
+    public get intelliSenseUpdateDelay(): number { return this.getAsNumber("intelliSenseUpdateDelay"); }
+    public get errorSquiggles(): string { return this.getAsString("errorSquiggles"); }
+    public get inactiveRegionOpacity(): number { return this.getAsNumber("inactiveRegionOpacity"); }
+    public get inactiveRegionForegroundColor(): string | undefined { return changeBlankStringToUndefined(this.getAsStringOrUndefined("inactiveRegionForegroundColor")); }
+    public get inactiveRegionBackgroundColor(): string | undefined { return changeBlankStringToUndefined(this.getAsStringOrUndefined("inactiveRegionBackgroundColor")); }
+    public get autocomplete(): string { return this.getAsString("autocomplete"); }
+    public get autocompleteAddParentheses(): boolean { return this.getAsBoolean("autocompleteAddParentheses"); }
+    public get loggingLevel(): string { return this.getAsString("loggingLevel"); }
+    public get autoAddFileAssociations(): boolean { return this.getAsBoolean("autoAddFileAssociations"); }
+    public get workspaceParsingPriority(): string { return this.getAsString("workspaceParsingPriority"); }
+    public get workspaceSymbols(): string { return this.getAsString("workspaceSymbols"); }
+    public get exclusionPolicy(): string { return this.getAsString("exclusionPolicy"); }
+    public get refactoringIncludeHeader(): string { return this.getAsString("refactoring.includeHeader"); }
+    public get simplifyStructuredComments(): boolean { return this.getAsBoolean("simplifyStructuredComments"); }
+    public get doxygenGeneratedCommentStyle(): string { return this.getAsString("doxygen.generatedStyle"); }
+    public get doxygenGenerateOnType(): boolean { return this.getAsBoolean("doxygen.generateOnType"); }
+    public get commentContinuationPatterns(): (string | CommentPattern)[] {
+        const value: any = super.Section.get<any>("commentContinuationPatterns");
+        if (this.isArrayOfCommentContinuationPatterns(value)) {
+            return value;
+        }
+        const setting = getRawSetting("C_Cpp." + "commentContinuationPatterns");
+        return setting.default;
+    }
+    public get isConfigurationWarningsEnabled(): boolean { return this.getAsString("configurationWarnings").toLowerCase() === "enabled"; }
+    public get preferredPathSeparator(): string { return this.getAsString("preferredPathSeparator"); }
+    public get updateChannel(): string { return this.getAsString("updateChannel"); }
+    public get isVcpkgEnabled(): boolean { return this.getAsString("vcpkg").toLowerCase() === "enabled"; }
```

Comment:
The setting is C_Cpp.vcpkg.enabled and not "vcpkg". I see it as a boolean and not a string type.